packages/core/src/utils/prepareEvent.ts

```diff
@@ -112,6 +113,54 @@ function applyClientOptions(event: Event, options: ClientOptions): void {
   }
 }
 
+/**
+ * Applies debug metadata images to the event in order to apply source maps by looking up their debug ID.
+ */
+export function applyDebugMetadata(event: Event, stackParser: StackParser): void {
+  const debugIdMap = GLOBAL_OBJ._sentryDebugIds;
```

Comment:
Where is this `_sentryDebugIds` actually set? Can't find it anywhere in the codebase...?

Reply:
Oh I should definitely add a comment explaining this --> 485b9810e16f1aa244b83ef3aa3c7ee0d9c3bc77